### PR TITLE
Unicode 14 files 20210602; updated script metadata

### PR DESCRIPTION
**Pull request by markusicu**
UCD/idna/emoji files from today jun02.
Many of the security/ files apparently get overwritten in the source data folder each time the generator tool is run.
Using updated script metadata from https://github.com/unicode-org/cldr/pull/1291
Setting Znamenny to Identifier_Type=Technical.

Updated Identifier*.txt files in https://corp.unicode.org/~book/incoming/markus/u14/20210602/security/

### CODE REVIEW

File: unicodetools/data/security/14.0.0/data/draft-restrictions.txt [hunk 1]
```diff
@@ -82,14 +82,14 @@
 082C          ;  ; Exclusion #      (ࠬ)  SAMARITAN VOWEL SIGN SUKUN
 082D          ;  ; Exclusion #      (࠭)  SAMARITAN MARK NEQUDAA
 16EE..16F8    ;  ; Exclusion # [11] (ᛮ..ᛸ)  RUNIC ARLAUG SYMBOL..RUNIC LETTER FRANKS CASKET AESC
-1700..170C    ;  ; Exclusion # [13] (ᜀ..ᜌ)  TAGALOG LETTER A..TAGALOG LETTER YA
-170E..1714    ;  ; Exclusion #  [7] (ᜎ..᜔)  TAGALOG LETTER LA..TAGALOG SIGN VIRAMA
-1720..1734    ;  ; Exclusion # [21] (ᜠ..᜴)  HANUNOO LETTER A..HANUNOO SIGN PAMUDPOD
+1700..1715    ;  ; Exclusion # [22] (ᜀ..᜕)  TAGALOG LETTER A..TAGALOG SIGN PAMUDPOD
```
Comment (markusicu) [inline unicodetools/data/security/14.0.0/data/draft-restrictions.txt:85]: FYI most of the data files in this folder seem to be *outputs*

File: unicodetools/data/security/14.0.0/data/source/removals.txt [hunk 2]
```diff
@@ -692,6 +692,9 @@ AB63; uncommon-use # LATIN SMALL LETTER UO
 \p{block=Ancient_Greek_Musical_Notation} ; obsolete
 \p{block=Phaistos_Disc} ; obsolete
 
+# 14.0
+\p{block=Znamenny_Musical_Notation} ; technical
```
Comment (markusicu) [inline unicodetools/data/security/14.0.0/data/source/removals.txt:696]: Here is a manual change in an input file. Before this, Znamenny symbols and marks got Identifier_Type=Recommended or Not_XID.

File: unicodetools/org/unicode/text/UCD/IdentifierInfo.java [hunk 3]
```diff
@@ -60,14 +58,14 @@
...
     static IdentifierInfo getIdentifierInfo() {
-	try {
-	    if (info == null) {
-		info = new IdentifierInfo();
-	    }
-	    return info;
-	} catch (final Exception e) {
-	    throw (RuntimeException) new IllegalArgumentException("Unable to access data").initCause(e);
-	}
+        try {
```
Comment (markusicu) [inline unicodetools/org/unicode/text/UCD/IdentifierInfo.java:61]: FYI  I untabified this file when I thought I needed to make changes here. Later I found that I could just add to the removals.txt file.

File: unicodetools/org/unicode/tools/Ids.java [hunk 4]
```diff
@@ -984,7 +984,7 @@ private static void showRadicalCompare() throws IOException {
 
     private static void showRadicalCompareTxt() throws IOException {
 
-        try (PrintWriter out = FileUtilities.openUTF8Writer(orgSettings.Output.GEN_DIR + "ids/", "radicalCompare.txt");
+        try (PrintWriter out = FileUtilities.openUTF8Writer(Settings.Output.GEN_DIR + "ids/", "radicalCompare.txt");
```
Comment (markusicu) [inline unicodetools/org/unicode/tools/Ids.java:987]: Apparently before I pushed my previous environment variable refactoring I fat-fingered "org" into the wrong window... oops!

**Review by macchiati — APPROVED** [review]
looks good to me

**Pull request merged**